examples/javascript/test/bidirectional/bidi.spec.js

```diff
@@ -0,0 +1,52 @@
+const { suite } = require('selenium-webdriver/testing');
+const assert = require("assert");
+const firefox = require('selenium-webdriver/firefox');
+const LogInspector = require('selenium-webdriver/bidi/LogInspector');
```

Comment:
windows often uses a case insensitive filesystem, while linux often does not. we should do use `require('selenium-webdriver/bidi/logInspector')` instead.

Reply:
Changes made. Thank you